src/components/Accounts/utils/ProxyTypeBadges.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+/* eslint-disable sort-keys */
+
+import React from 'react';
+
+interface ProxyTypeBadgesProps {
+	text: string;
+}
+
+const ProxyTypeBadges: React.FC<ProxyTypeBadgesProps> = ({ text }) => {
+	const getStyles = () => {
```

Comment:
it can a static function
you can write this outside the component and get text from the param
so if component rerendered this function doesn't have any effects